include/spl/sys/sysmacros.h

```diff
@@ -115,30 +115,36 @@
 #endif
 
 /* Dtrace probes do not exist in the linux kernel */
```

Comment:
Maybe we should change this comment to talk about how we've hooked these "dtrace" probes up to kprobes?

Reply:
Yes, that would be more helpful.